charts/incubator/miniflux/values.yaml

```diff
@@ -28,24 +15,46 @@ persistence:
     enabled: true
     mountPath: "/config"
 
-probes:
-  liveness:
+cnpg:
+  main:
     enabled: true
-    custom: true
-    spec:
-      initialDelaySeconds: 30
-      periodSeconds: 10
-      timeoutSeconds: 1
-      failureThreshold: 3
-      httpGet:
-        path: /healthcheck
-        port: 8080
-
-postgresql:
-  enabled: true
-  existingSecret: "dbcreds"
-  postgresqlDatabase: miniflux
-  postgresqlUsername: miniflux
+    user: miniflux
+    database: miniflux
 
 portal:
-  enabled: true
+  open:
+    enabled: true
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
```

Comment:
```suggestion
```

Those are the default

Reply:
Removed in 2df4830510a0a251a34b219292c587be1b0e5303.